frontend/src/main.tsx

```diff
@@ -1,34 +1,23 @@
 import { StrictMode } from "react";
 import ReactDOM from "react-dom/client";
 import { RouterProvider, createRouter } from "@tanstack/react-router";
-import clientMetadata from "../client-metadata";
 // Import the generated route tree
 import { routeTree } from "./routeTree.gen";
 import reportWebVitals from "./reportWebVitals.ts";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-import { BrowserOAuthClient } from "@atproto/oauth-client-browser";
+import { AuthManager } from "./auth.ts";
 
-export const oauthClient = new BrowserOAuthClient({
-  handleResolver: "https://bsky.social",
-  clientMetadata: __DOMAIN__
-    ? clientMetadata(__DOMAIN__)
-    : {
-      client_id: "http://localhost?scope=atproto%20transition%3Ageneric",
-      redirect_uris: ["http://127.0.0.1:5173/"],
-      scope: "atproto transition:generic",
-      token_endpoint_auth_method: "none",
-    },
-  allowHttp: true,
+const authManager = new AuthManager("privi.dwelf-mirzam.ts.net", () => {
```

Comment:
can we have this take in a domain rather than use a hard coded one? just to make it easier as we move around where we run / host this

Reply:
yeah was gonna do it in a followup.